modules/templates/helper.go

```diff
@@ -971,20 +972,35 @@ type remoteAddress struct {
 	Password string
 }
 
-func mirrorRemoteAddress(ctx context.Context, m repo_model.RemoteMirrorer) remoteAddress {
+func mirrorRemoteAddress(ctx context.Context, m *repo_model.Repository, remoteName string) remoteAddress {
 	a := remoteAddress{}
+	if !m.IsMirror {
+		return a
+	}
```

Comment:
@lunny  I saw an issue here, the push mirrors URL are no longer displayed on the UI

![image](https://user-images.githubusercontent.com/98962410/173442668-8188b608-3243-4250-bfcc-11941da50ec2.png)

I think we should return  empty remoteAddress when the repo  is a mirror not the opposite

``` go
	if m.IsMirror {
		return a
	}
```